docs/guide/security_checklist.md

```diff
@@ -115,9 +115,22 @@ Use the following checklist to ensure the Roles and Policies are secure:
 
 ## Underlying stack
 
+Once you have properly configured secure user roles and permissions, to avoid exposing your application to DDOS vulnerabilities and other, yet unknown security threats, make sure that you do the following:
```

Comment:
```suggestion
Once you have properly configured secure user roles and permissions, to avoid exposing your application to any DDOS vulnerabilities or other yet unknown security threats, make sure that you do the following:
```
Being very clear that there are no _known_ such vulnerabilities.